src/python/deepops/deps.py

```diff
@@ -0,0 +1,11 @@
+import os
+import subprocess
+from .repo import clone_repo, local_repo_path
+
+
+def run_deepops_setup():
+    clone_repo()
```

Comment:
Instead of cloning the repo, I think you should just run the equivalent of `curl -sL git.io/deepops | bash -s -- TAG`.  The setup script clones the repo, but also ensure the system is setup right.

Here's some python to do that:
```
import os
import requests
from pathlib import Path
import subprocess
import tempfile

def self_install(install_directory=None, tag='19.10'):
    # Create destination directory
    if install_directory:
        try:
            Path(install_directory).mkdir(parents=True, exist_ok=True)
        except Exception as e:
            print(e)
            exit(e.errno)

        os.chdir(install_directory)

    # Get DeepOps setup script
    res = requests.get('https://git.io/deepops')
    if len(res.text) <= 0:
        print('Did not get setup script')
        exit(1)

    # Create temporary copy of script
    f = tempfile.NamedTemporaryFile(dir='.', suffix='.sh')
    Path(f.name).write_text(res.text)

    subprocess.call(['bash', '-x', f.name, tag])
```

Reply:
So, I made this choice for a few reasons.

1. I wanted to give the user the option to decouple cloning the repo from installing the DeepOps dependencies. I suppose this could be a flag (like `--no-setup`).
1. I don't really like the https://git.io/deepops redirect, because it always points to the version of the script on `master`.
    It essentially always makes the user run the development version of the setup script. I'd prefer to be able to specify a release version even at this stage, so that we can put the user on some "known good" path from the start.
    If we could automatically have some set of redirects like https://git.io/deepops-19.10 then this would be less of an issue, but that seems like a pain to maintain.

That said, I'm open to being convinced on either of those points. :) I'm curious what @dholt and @ryanolson have to say.